### ISSUE TITLE
License of the theme?

### ISSUE THREAD

**Issue by hartwork** [CMoH/grub2-theme-gentoo#1]
Hi!

I learned about this theme from @N8Fear's [blog comment over here](https://blog.hartwork.org/?p=2566#comment-707207). I am wondering what the effective license is.
[The ebuild](https://github.com/CMoH/portage-overlay/tree/master/media-gfx/grub2-theme-gentoo) you link to in `README.md` says it's GPL v3 (impliying that version and that only) but I did not find anything backing that up. [The tarball](https://non7top.googlecode.com/files/gentoo.tar.bz2) you link to does not seem to contain any license information by itself. The source listing of the [hosting Google Code project](https://code.google.com/p/non7top/source/browse/) does not seem to contain any related source files or licensing notes, either.
Please help me understand the license situation.

Thanks, Sebastian

**Comment by CMoH** [comment]
I have organized and embellished the theme from the quoted source for personal use, but next I decided to upload all that to github (both the theme and my personal portage overlay) for others to comment/use/improve. I don't think portage is able to install stuff with unclear license, so GPLv3 seemed a good placeholder, and well.. forgot about it.

I reached that download link from another page, where they guy described that it's based on the ArchLinux grub2 theme, but I was unable to find that page when creating the ebuild. I have no clue what license he might place on it, but lacking any I assume it's simply public domain. We could try and reach him using the google code project and clarify the situation.

**Comment by hartwork** [comment]
> but lacking any I assume it's simply public domain

When something does not mention an explicit license, the only _safe_ assumption is "all rights reserved" I'm afraid.

> We could try and reach him using the google code project and clarify the situation.

Could you do that?

To be fair I should mention, the icons folder is yet another story. But let's focus on the main theme for now.

**Comment by CMoH** [comment]
I can try and contact the guy, but I have to ask to what purpose are you asking about the license.

"Safe assumption" in relation to ... what? Redistributing the theme? Being sued over its usage or modifications?

**Comment by hartwork** [comment]
Hi!

> I have to ask to what purpose are you asking about the license.

Initially, it was a pure question: I wanted to understand the current situation.
I'm a Gentoo developer with emphasis on Software libre so I care about the options that users of Gentoo with focus on free-as-in-speech have.

Right now the question is, if (excluding the distro icons again) the theme _itself_ can still be put on solid, libre ground. For that, the license of all parts contained would need to be clear.

To give an example the "g" logo in the background looks like the ["Vector version plain by Matteo Pescarin" on the Project:Artwork/Artwork Gentoo wiki page](https://wiki.gentoo.org/wiki/Project:Artwork/Artwork#Vector_version_.22g.22_logo). If it is, the person who embedded that image in another one would have needed to put the resulting image under the [CC-BY-SA/2.5 license](https://creativecommons.org/licenses/by-sa/2.5/) as well, since that is what the Share-Alike aspect (SA) is demanding. So that fact would need to be brought to the attention of the person who did the initial embedding, he or she would have to agree that the resulting image can be used under CC-BY-SA/2.5, if possible (..). If not possible or if he/she doesn't give that agreement, that image wouldn't be usable to anyone legally, including him/her.

> "Safe assumption" in relation to ... what?

Safe assumption in relation to copyright in general _any_ relation more or less.

> Being sued over its usage or modifications?

I'm not here to sue anyone and I could not either, since this theme does not contain any work by me or that I would have copyright of.

By the way I have [added an ebuild](https://github.com/gentoo/grub2-themes-overlay/tree/master/grub2-themes/cmoh-gentoo) to the [new grub2-themes overlay](https://github.com/gentoo/grub2-themes-overlay), now.

I hope that answers your questions. I know that licensing is little fun.

**Comment by non7top** [comment]
Guys, I see that file is 7 years old and unfortunately I can't recall where I got it, most likely somewhere on gentoo forums I think. Either way I'm not the author of any data in the tarball nor do I claim any copyright on that data. 

Quick googling revealed this: http://forums.gentoo.org/viewtopic-t-715590.html

**Comment by CMoH** [comment]
Thank you for your reply, non7top.

Apparently user cheater1034 on the gentoo forum found the theme at http://omploader.org/veWtn (which now yields 404). The forum mentioned by him does contain a theme file on page 2, but I haven't found any working links towards the graphics used. To be fair, the original tar.bz2 does contain the archdark.png files and such.

Apparently a license discussion occured on the same forum, at https://bbs.archlinux.org/viewtopic.php?pid=556817#p556817

So, where do we go from here, hartwork? Contact cheater1034 on gentoo forums?

**Comment by hartwork** [comment]
Hi, sorry for the delay!

In that post that @non7top linked to, cheater1034 says 

> The guide for grub2-graphical: http://bbs.archlinux.org/viewtopic.php?id=56576 
> [..]
> The gentoo grub2 theme there is the one posted on the last page on that forum, except it has changed logos for gentoo (gentoodark and gentoocenter instead of arch*)

My understanding is that "last page on that forum" refers to the `bbs.archlinux.org` link he shared above, to the last page at the time of his posting, which is "Tue Nov 25, 2008 3:54 am".
Dates on page two on that thread already go beyond that date, so I suppose he refers to page 2, since page 1 would hardly be called "the last page". Even for a second page, "the last page" seems an unlikely wording to me. On that second page, I'm not sure what his source may be.

To summarize we see to have this chain of re-use now, backwards in time:
- CMoH (made changes)
- non7top (uploaded, no changes)
- cheater1034 (made changes)
- _to be discovered_

> So, where do we go from here, hartwork? Contact cheater1034 on gentoo forums?

Sounds good.
